homepage/templates/homepage/edit.html

```diff
@@ -0,0 +1,7 @@
+<form method="post" novalidate>
```

Comment:
Do not copy paste, use same file. Even if there are minor changes, you can use same file